packages/site/src/layout/SearchBox.tsx

```diff
@@ -101,7 +101,7 @@ export const SearchBox = ({
             </Content>
           )}
           {filteredDesignList.length > 0 && (
-            <Box padding={{ top: "largest" }}>
+            <Box>
```

Comment:
Do we actually still need this to be a Box if we're not using it to apply any styling? (I guess it could stick around and if we need it, there's no real negative impact of keeping it here.)

Reply:
Uhmmm tbh dunno! It doesn't seem to break anything when removed... so it's probably okay? 😅

Reply:
If there are uses of `Box` that aren't currently providing any benefits styling wise I would advocate for removing them

Reply:
@Beccaneer if this is the case I think it is something we should clean up at some point, not worth blocking this PR though